lib/prmd/commands/combine.rb

```diff
@@ -79,7 +79,13 @@ def self.combine(paths, options = {})
       base = Prmd::Template.load_json('combine_head.json')
       schema = base['$schema']
       meta = {}
-      meta = Prmd.load_schema_file(options[:meta]) if options[:meta]
+      if filename = options[:meta]
+        meta = Prmd.load_schema_file(filename)
+        if meta.nil?
```

Comment:
I suspect we might want this to `meta.empty?` instead of `meta.nil?`, what do you think?